packages/at_secondary_server/test/enroll_verb_test.dart

```diff
@@ -799,6 +909,8 @@ void main() {
       // First Invalid request
       String enrollmentRequest =
           'enroll:request:{"appName":"wavi","deviceName":"mydevice","namespaces":{"wavi":"r"},"otp":"123","apkamPublicKey":"dummy_apkam_public_key"}';
+      print(EnrollParams.fromJson(
```

Comment:
remove this print

Reply:
change made in [fix: make verbParams optional param from fetchEnrollments()](https://github.com/atsign-foundation/at_server/pull/1846/commits/ffebc36ffdb07cc4133207cbdb68c9063c6aa7e0)